testing/python/metal/test_metal_threadsync_simdgroup_store.py

```
@@ -0,0 +1,467 @@
+
+
+@tilelang.jit
+def _access_ptr_waw(out, stride, threads=64):
+    """Two sequential simdgroups store overlapping 8x8 shared tiles.
+
+    The first tile covers rows 1..8 and the second covers rows 8..15, so
+    row 8 requires a barrier between the stores. The final load keeps the
+    shared-memory writes observable in generated code.
+    """
+    out: T.Tensor((8, 16), "float32")
+    with T.Kernel(1, threads=threads) as (bx,):
```

Ruff (0.16.1)

[warning] Unpacked variable `bx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)